tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -115,7 +115,11 @@ internal TracerSettings(IConfigurationSource? source, IConfigurationTelemetry te
                                 _ when x.ToBoolean() is { } boolean => boolean,
                                 _ => ParsingResult<bool>.Failure(),
                             },
-                            getDefaultValue: () => false,
+                            getDefaultValue: () =>
+                            {
+                                var profilingSsiDeployed = config.WithKeys(ContinuousProfiler.ConfigurationKeys.SsiDeployed).AsString();
```

Comment:
We're enabling profiling by default in SSI?! 😨 I mean, we're _not_ enabling SSI by default in SSI, so I don't know what the implications of this being wrong would be...

Reply:
> We're enabling profiling by default in SSI?! 😨 I mean, we're _not_ enabling SSI by default in SSI, so I don't know what the implications of this being wrong would be...

Good catch!
This code was the initial implementation where the profiler was enabled by default if installed via SSI.
Now, we only enable the profiler if the env var contains "profiler".

Reply:
I'm just wondering, is it useful to explicitly have a difference between `ProfilingSsiDeployed` and `ProfilingEnabled`